src/utils/hsdis/binutils/hsdis-binutils.c

```diff
@@ -61,14 +61,10 @@
 
 #include <bfd.h>
 #include <dis-asm.h>
+#include <stdbool.h>
 
 #include "hsdis.h"
 
-#ifndef bool
```

Comment:
I'm a little worried about this change. hsdis may really need an int here. If that turns out to not be the case then I'll retract my concerns

Reply:
I have verified this change locally, include build hsdis.so and check the functional with command java -XX:+UnlockDiagnosticVMOptions -XX:+PrintAssembly -version. The verified show this change for hsdis.so work normally.

Reply:
Ok, sounds good

Reply:
Thanks for the review.